github/orgs_teams_test.go

```diff
@@ -508,7 +509,34 @@ func TestOrganizationsService_ListPendingTeamInvitations(t *testing.T) {
 		testMethod(t, r, "GET")
 		testFormValues(t, r, values{"page": "1"})
 		testHeader(t, r, "Accept", mediaTypeOrgMembershipPreview)
-		fmt.Fprint(w, `[{"id":1}]`)
+		fmt.Fprint(w, `[
```

Comment:
@gmlewis, @shurcooL  I agree I made the test case big with the full test data. I have used all values that are returned by this API call, i felt test coverage will be better that way. I was not sure which field to omit, as JSON will return all of them. 
Is there a guideline to choose some of these fields from from the response?